src/olympia/constants/permissions.py

```diff
@@ -15,8 +15,6 @@
 # access to anything.
 SUPERPOWERS = AclPermission('*', '*')
 
-# Can access admin-specific tools.
-ADMIN_TOOLS = AclPermission('Admin', 'Tools')
```

Comment:
This permission is used 183 times in tests so you can't really remove it without changing them all (look for `Admin:Tools`)  (we should probably have some validation to stop missing permissions being assigned in `grant_permission`... but we don't)

Reply:
The fact that all tests pass makes that kind of useless anyway. The permission is not used anywhere (outside tests), and tests use the constant, so why do we need to keep a dead constant 🤷 ?

Reply:
No, it's that the test used for access to those views is `ANY_ADMIN` so `Admin:Tools` passes.  I'm not saying we should keep a dead constant, I'm saying you'll have to change all the usage to another permission.

Reply:
Ok I'll add it back, but I don't think it's particularly helpful. The tests set a permission that is unused, regardless of whether a dead constant is defined or not.

Reply:
46243ea51b